### ISSUE TITLE
Cron job failure

### ISSUE THREAD

**Issue by Larx** [H2CK/oidc#3]
On Nextcloud 23.0.4 with PHP8.0, the Nextcloud cron job fails (using the app version as provided from the Nextcloud store). I assume from the output that it is some cleanup job of the OIDC app, as I have no other OpenID app installed. (I start the cron job with a systemd-timer.)

```
Apr 29 11:56:57 server systemd[1]: Started Nextcloud cron.php job.
Apr 29 11:56:58 server php[61717]: ArgumentCountError: Too few arguments to function OCA\OIDCIdentityProvider\BackgroundJob\CleanupExpiredTokens::__cons>
Apr 29 11:56:58 server php[61717]: Stack trace:
Apr 29 11:56:58 server php[61717]: #0 /srv/www/nextcloud/lib/private/BackgroundJob/JobList.php(291): OCA\OIDCIdentityProvider\BackgroundJob\CleanupExpir>
Apr 29 11:56:58 server php[61717]: #1 /srv/www/nextcloud/lib/private/BackgroundJob/JobList.php(225): OC\BackgroundJob\JobList->buildJob()
Apr 29 11:56:58 server php[61717]: #2 /srv/www/nextcloud/cron.php(144): OC\BackgroundJob\JobList->getNext()
Apr 29 11:56:58 server php[61717]: #3 {main}
Apr 29 11:56:58 server systemd[1]: nextcloudcron.service: Main process exited, code=exited, status=1/FAILURE
```

**Comment by H2CK** [comment]
Thanks for reporting this bug. I can confirm this unwanted behavior.
In generally this BackgroundJob only cleans expired tokens from the database. So the main functionality is not affected.
I try to provide a fix soon.

**Comment by H2CK** [comment]
Fixed with release 0.1.1.
